src/main/scala/ai/privado/languageEngine/java/tagger/source/IdentifierTagger.scala

```diff
@@ -69,12 +69,25 @@ class IdentifierTagger(cpg: Cpg) extends ForkJoinParallelCpgPass[RuleInfo](cpg)
       addRuleTags(builder, identifier, ruleInfo)
     })
 
-    // Step 2.1 --> contains step 2.2, 2.3
+    // Step 2.1
+    tagMethodReturn(builder, rulePattern, ruleInfo)
+
+    // Step 2.2 --> contains step 2.3, 2.4
     tagObjectOfTypeDeclHavingMemberName(builder, rulePattern, ruleInfo)
+  }
 
+  /** Tag return object of regex matching method Step 2.1
+    */
+  private def tagMethodReturn(builder: BatchedUpdate.DiffGraphBuilder, rulePattern: String, ruleInfo: RuleInfo):  Unit = {
+    val methodReturnMatches = cpg.method(rulePattern).repeat(_.astChildren)(_.emit).filter(_.isReturn).l
```

Comment:
The current rule which we have in place targets identifiers, if here we are looking to target `get<PII>`. we need to update the query to `cpg.method("get"+rulePattern)`, so that we are not losing coverage here